nnvm/python/nnvm/frontend/onnx.py

```diff
@@ -611,6 +611,46 @@ def _impl_v1(cls, inputs, attr, params):
                 'axis': ('axis', 1),
             })(inputs, attr, params)
 
+class ConstantFill(OnnxOpConverter):
+    """ Operator converter for ConstantFill.
+    """
+    @classmethod
+    def _impl_v1(cls, inputs, attr, params):
+        is_full = True
+        num_inputs = len(inputs)
+        if 'shape' in attr:
+            if num_inputs > 0:
+                raise ImportError(
+                    "Can't set shape and input tensor at a time")
+            shape = attr.pop('shape')
+        else:
+            if num_inputs == 0:
+                raise ImportError(
+                    "Either shape attribute or input should be set")
```

Comment:
Does this code work when `num_inputs` is greater than 1? If not, I'd suggest raising the error also when num_inputs is not 1.